### DISCUSSION TITLE
Download failed for Edge

### DISCUSSION THREAD

**Discussion by xjeffesilva** [aclap-dev/video-downloadhelper#869] [Help and Q&A]
### Your question

Good morning,

In the Egde browser, it is not possible to download.

Below are the bug details

{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "beta",
  "page_url": "https://oficinadotrader.alpaclass.com/c/cursos/9eVKMk?lessonSlug=yODWQ6",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 7 times\r\n[in#0 @ 0000022920277880] Error opening input: Invalid data found when processing input\r\nError opening input file https://subplaylist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiJ9.eyJrZXkiOiJiYjA0NDMzZi0yNDQ4LTRmMzUtOGNjOC01ZDk0NjE1NWZjMjUiLCJzZXNzaW9uS2V5IjoiYjM3dXNqcnMiLCJwcm9kdWNlcktleSI6IjI0ODE2ODEwIiwicHJvZHVjZXJFbWFpbCI6ImVkdXp6QG9maWNpbmFkb3RyYWRlci5jb20uYnIiLCJjb25zdW1lckVtYWlsIjoiamVmZl9hX3NpbHZhQGhvdG1haWwuY29tIiwiaW1wcmludCI6dHJ1ZSwic3RvcmFnZSI6InIyIiwicGwiOiIxMDgwcF9wbGF5bGlzdC5tM3U4IiwiaWF0IjoxNzE4MjgyNTMzLCJleHAiOjE3MTg0NTUzMzN9.TPMPhzV77JEIfP0sYW9iPAeiWdZSUxDK9OM5eia1_AE.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_7469997666374655",
    "error": "coapp_failure",
    "id": "coapp_failure_fe031051-51a4-4991-a604-be7330234984"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: microsoft
channel: beta
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 Edg/125.0.0.0
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 7
}

**Comment by Season4episode10** [comment]
Site is password protected, rules out others testing the site.

**Reply by xjeffesilva** [reply]
I'll download it via Firefox. Thanks for the reply.

**Reply by Mattmjs** [reply]
It may not work in Firefox , seeing that there was another report like this one 

https://github.com/aclap-dev/video-downloadhelper/discussions/694
